GCRCatalogs/catalog_configs/photoz_mock_magnitudes_with_errors_for_cosmoDC2_v1.1.4_image.yaml

```diff
@@ -3,6 +3,7 @@
 subclass_name: photoz_magerr.PZMagErrCatalog
 catalog_name: photoz_mock_magnitudes_with_errors_for_cosmoDC2_v1.1.4_image
 base_dir: ^/xgal/cosmoDC2/addons/photoz_v1.1.4/IMAGE/10_year_error_estimates
+filename_pattern: 'z_(\d)\S+withmask.healpix_(\d+)_magwerr\.h5'
```

Comment:
Is this line still needed given that you have edited the default `FILE_PATTERN`?

Reply:
Ah, good catch, I put that in before remembering I could change the default and forgot to take it out.  I just removed it and pushed that change.